Automattic/jetpack
# DNA: Deprecate tracks/client lib

Deprecates tracks/client lib moving it to the compat package
Adds deprecation tests for libs and global defined functions

#### Testing instructions:
* run the tests
* try to use [woocommerce-services](https://github.com/Automattic/woocommerce-services/blob/5fb39adb9814bef92ea80374fe989207ca961e47/classes/class-wc-connect-tracks.php#L69) 
* add this snippet to your test site
```
add_action( 'plugins_loaded', function() {
    jetpack_tracks_get_identity( 1 );
}, 99 );
```
* make sure you get a deprecation notice
